### ISSUE TITLE
add Geoportal 2: ISOK hillshade imagery

### ISSUE THREAD

**Issue by matkoniecz** [osmlab/editor-layer-index#476]
When **requesting the addition of an imagery layer**, please provide the following information. Otherwise, delete the text and type your issue.

## URL of the imagery
(required)

## Two letter country code
PL

## Desired layer name
ISOK hillshade

## License
(required)<br/>
See [FAQ.md](FAQ.md#what-imagery-licenses-are-compatible-with-this-index) for information
about which licenses are compatible with this index.

effectively PD - see https://wiki.openstreetmap.org/wiki/Geoportal.gov.pl

Available in JOSM ( https://josm.openstreetmap.de/wiki/Maps ) so it should be OK (if it is not OK then it should be investigated quickly as it would make large part of map data in Poland a copyright violation)

## Is the license compatible with OSM?
(required)

see above

## Attribution (text, URL, required?)
(optional)

no

## Icon
(optional)

## Bounding polygon
(if possible, provide a bounding polygon where the imagery is valid, e.g. as a GeoJSON or OSM file.)<br/>
See [FAQ.md](FAQ.md#how-can-i-draw-a-bounding-polygon) for information about how to draw a bounding polygon.

should be importable from JOSM

**Comment by matkoniecz** [comment]
This layer has an extreme detail - not only objects like streams and peaks are visible, it is possible to map paths in forests based on a terrain shape.

**Comment by grischard** [comment]
I didn't know JOSM could set HTTP headers like that!

It's a pity, as Simon said in the other ticket, that it doesn't support EPSG:3857, which means iD won't be able to use it. Would the Polish community be interested in running a mapproxy instance?

**Comment by simonpoole** [comment]
@grischard iD suppoerts EPSG:4326 (just as Vespucci does) and looking at the JOSM entry it would seem to be supported.

**Comment by tyrasd** [comment]
> iD supports EPSG:4326

yes, but only for WMS services, and not for WMTS/TMS layers. Also, this one seems to only support [`EPSG:2180` ](http://epsg.io/2180). At least this is an example of a tile which JOSM actually loads: http://mapy.geoportal.gov.pl/wss/service/WMTS/guest/wmts/ISOK_CIEN?SERVICE=WMTS&REQUEST=GetTile&VERSION=1.0.0&LAYER=ISOK_Cien&STYLE=default&FORMAT=image/jpeg&tileMatrixSet=EPSG:2180&tileMatrix=EPSG:2180:11&tileRow=1357&tileCol=1837
